server/auth/authController.js

```diff
@@ -16,33 +16,35 @@ module.exports = function(passport){
   clientSecret: localServices.facebook.secret,
   callbackURL: client.facebookAuth.callbackURL
```

Comment:
Can you let us know what callback URL we need to manually hard code for now when testing?